tests/basictests/dsp-operator.sh

```diff
@@ -120,6 +141,34 @@ EOF
     os::cmd::try_until_not_text "curl -s -k -H 'Authorization: Bearer ${SA_TOKEN}' https://${ROUTE}/apis/v1beta1/runs/${RUN_ID} | jq '" "null" $odhdefaulttimeout $odhdefaultinterval
 }
 
+function create_experiment_run() {
```

Comment:
as far as i can tell, this is unused?

Reply:
Done, updated the function calls to have one for this function as well. Has been tested as part of https://github.com/opendatahub-io/data-science-pipelines-operator/pull/95, function works as expected - creates a run for the experiment defined in one of the earlier functions.